docs/languages/Csharp.md

```diff
@@ -15,6 +15,15 @@ There are special use-cases that each language supports; this document pertains
 
 Sometimes you want to serialize the data models into JSON. In order to do that use the preset `CSHARP_JSON_SERIALIZER_PRESET`
 
+**Important Note:**
+The following dependencies used in the serializer and deserializer functionality are .NET specific dependencies.
+
+```
+System.Text.Json
+System.Text.Json.Serialization
+System.Text.RegularExpressions
+```
```

Comment:
Would it make sense to also give direct links to the dependencies? Such as https://github.com/asyncapi/modelina/pull/596/files#diff-9dd59a8eea1ff041d2dd3864867d10d85bf26318ff2848e23245a7f4a113d67cR67-R68 ?

Reply:
Yeah, It's a nice idea I will update it.